### ISSUE TITLE
Cloud "Save Kubeconfig" doesn't work with RBAC enabled cluster

### ISSUE THREAD

**Issue by timja** [Azure/vscode-aks-tools#4]
```
'Tim.Jacomb@HMCTS.NET' with object id '****' does not have authorization to perform action 'Microsoft.ContainerService/managedClusters/accessProfiles/listCredential/action' over scope '/subscriptions/***/resourceGroups/demo-00-rg/providers/Microsoft.ContainerService/managedClusters/aks-demo-cluster-00/accessProfiles/clusterUser'.
```

https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#azure-kubernetes-service-cluster-user-role

The built in role gives:
"Microsoft.ContainerService/managedClusters/listClusterUserCredential/action"

Whereas this vscode extension is trying to use:
"Microsoft.ContainerService/managedClusters/accessProfiles/listCredential/action"

**Comment by itowlson** [comment]
@timja Sorry for dropping the ball on this one and for so long, and thank you for pointing me at the API I should be using.  Can you clarify how to reproduce this issue?  I created a cluster with RBAC but Save Kubeconfig seems to be working okay for me.  Does this relate to the permissions of the logged-in Azure user (i.e. AAD RBAC on the Azure object) rather than RBAC being enabled on the cluster itself (i.e. Kubernetes RBAC on the Kubernetes resources)?  Thanks!

**Comment by timja** [comment]
> @timja Sorry for dropping the ball on this one and for so long, and thank you for pointing me at the API I should be using. Can you clarify how to reproduce this issue? I created a cluster with RBAC but Save Kubeconfig seems to be working okay for me. Does this relate to the permissions of the logged-in Azure user (i.e. AAD RBAC on the Azure object) rather than RBAC being enabled on the cluster itself (i.e. Kubernetes RBAC on the Kubernetes resources)? Thanks!

Yes it relates to your azure rbac role, if you only have cluster user role it doesn’t work

**Comment by itowlson** [comment]
@tatsinnit can you take a look at this please?

**Comment by Tatsinnit** [comment]
💡Hiya @timja ,  I tried to repro this, but it **works** for me, I wonder if this could be double check or specific steps to repro can be shared:

Here is what I did, for the successful scenario:

* Created `aks cluster` via portal default version of cluster I used was `1.15`.
* Once resource is all done: I went to `Access Control` here: 

<img width="270" alt="Screen Shot 2020-06-17 at 9 56 53 AM" src="https://user-images.githubusercontent.com/6233295/84832219-fa4eeb80-b080-11ea-93aa-d402b01fb2a3.png">

* Then Selected: `Add Role Assignment` and selected `User Role`

<img width="991" alt="Screen Shot 2020-06-17 at 9 58 16 AM" src="https://user-images.githubusercontent.com/6233295/84832432-5dd91900-b081-11ea-9fca-e1b1ede99c6e.png">


* When I run Save Kubeconfig, I get below dialog and kubeconfig gets saved.

<img width="551" alt="Screen Shot 2020-06-17 at 10 01 16 AM" src="https://user-images.githubusercontent.com/6233295/84832551-97118900-b081-11ea-9919-760e501855b5.png">

Thanks heaps,

fyi: cc: @itowlson

**Comment by Tatsinnit** [comment]
💡 **Update\To keep everyone in loop**: This issue will be resolved when the next release will go out, [this PR](https://github.com/Azure/vscode-aks-tools/pull/32) was merged to cater the issue. Keeping this to open state until next release. 

📔 This issue was only reproducible to the `aks clusters` where users don't `own` but only have `user` role for the cluster. Thank you Ivan for the Merge et. al. = )

**Comment by Tatsinnit** [comment]
💡  This issue is fixed with the release 0.0.7, thanks.
